### PR TITLE
Update SRS to reflect DRD changes

### PR DESCRIPTION
**Pull request by mocsharp**
### Description
Update design specification to reflect recent DRD #19 changes.

### Status
**Ready**

### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] All tests passed locally by running `./src/run-tests-in-docker.sh`.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [ ] User guide updated.
- [ ] I have updated the changelog

### CODE REVIEW

File: guidelines/diagrams/mig-acr.txt [hunk 1]
```diff
@@ -1,38 +1,58 @@
 # Created at https://sequencediagram.org/
-title MONNAI Informatics Gateway - ACR
+title MONNAI Informatics Gateway - SCP
```
Comment (hshuaib90) [inline guidelines/diagrams/mig-acr.txt:2]: typo: MONAI

File: guidelines/diagrams/mig-scp.txt [hunk 2]
```diff
@@ -2,40 +2,64 @@
 title MONNAI Informatics Gateway - SCP
```
Comment (hshuaib90) [inline guidelines/diagrams/mig-scp.txt:2]: typo: MONAI

**Comment by hshuaib90** [comment]
Thanks for this great work Victor. I think one of the key services missing is alerting/notification. This can be done via email (see page 6 "Sending automated email" https://s3-eu-west-1.amazonaws.com/comms-mat/Training-Materials/Guidance/applicationsguide.pdf) and/or perhaps some configurable generic POST API.

File: guidelines/diagrams/mig-export.txt [hunk 3]
```diff
@@ -1,38 +1,66 @@
...
 
+EW->PS: Subscribe to new export task events
+activate EW
+EW<--PS: Response
+
+PS->EW: Notify new export task event
 activate EW
-EW->EA: Query for pending export tasks
+
+EW->EA: Download results
```
Comment (joshliberty) [inline guidelines/diagrams/mig-export.txt:35]: @mocsharp I was under the impression that the MIG will talk directly to the storage layer to download payloads (having been given a reference in the export event notification) - and not go through the MWM.

**Review by joshliberty — COMMENTED** [review]
All in all looks good - the one comment I have is about the export & result download.

**Review by GreySeaWolf — APPROVED** [review]

File: guidelines/diagrams/mig-export.txt [hunk 4]
```diff
@@ -1,38 +1,66 @@
-# Created at https://sequencediagram.org/
-title MONNAI Informatics Gateway - Export
```
Comment (jonanswer) [inline guidelines/diagrams/mig-export.txt:2]: MONAI

File: guidelines/diagrams/mig-export.txt (hunk 3 above)
Comment (jonanswer) [reply guidelines/diagrams/mig-export.txt:35]: agree on this point

**Review by jonanswer — APPROVED** [review]

**Comment by mocsharp** [comment]
> Thanks for this great work Victor. I think one of the key services missing is alerting/notification. This can be done via email (see page 6 "Sending automated email" https://s3-eu-west-1.amazonaws.com/comms-mat/Training-Materials/Guidance/applicationsguide.pdf) and/or perhaps some configurable generic POST API.

Do we want a standalone service that aggregates events from all subsystems or shall each subsystem manage its own?
@dbericat

**Comment by dbericat** [comment]
> > Thanks for this great work Victor. I think one of the key services missing is alerting/notification. This can be done via email (see page 6 "Sending automated email" https://s3-eu-west-1.amazonaws.com/comms-mat/Training-Materials/Guidance/applicationsguide.pdf✎ EditSign) and/or perhaps some configurable generic POST API.
> 
> Do we want a standalone service that aggregates events from all subsystems or shall each subsystem manage its own? @dbericat

IMO 1 standalone service works best. Avoids duplication of effort and keeps the sub-systems leaner. 

But by design, we are not expecting every adopter to use the whole system. So, each sub-system should be able to package/use that micro-service/component/library locally (or externally).

What do others think?

File: guidelines/diagrams/mig-export.txt (hunk 3 above)
Comment (mocsharp) [reply guidelines/diagrams/mig-export.txt:35]: Good catch! Thanks

**Pull request merged**